packages/nimble-components/src/select/index.ts

```diff
@@ -216,6 +216,14 @@ export class Select
     private _value = '';
     private forcedPosition = false;
     private openActiveIndex?: number;
+    private readonly selectedOptionObserver: MutationObserver;
```

Comment:
Betting the PR failures have to do with constructor timing (propertyChange callbacks for initialized props run before contructors). Need to capture notes on it but instead initialize here:

```
private readonly selectionOptionsObserver = new MutationObserver(() => {this.updateDisplayValue()});
```

Reply:
I actually had to make its type `MutationObserver | undefined`, even when defined in this way, because of the issue you raise. Which also means using the null check operator where I access `selectedOptionObserver`.